sync_labels.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+import sys
+import os
+import argparse
+from github import Github, GithubException
+
+"""
+Sync Github Labels
+
+Synchronize labels idempotently between two Github repositories.
+"""
+
+
+def sync_labels(src_repo, dest_repo, dry_run, delete):
+    """
+    Main subroutine: sync labels between source/destination Github repos
+
+    :param src_repo: full repo name for source of labels
+    :param dest_repo: full repo name for destination of label actions
+    :param dry_run: do a dry run, do not perform the actions
+    :param delete: delete labels that are in destination but not in source
+    """
+
+    # Get PyGithub objects representing repos
+    g = Github(get_access_token())
+    s = g.get_repo(src_repo)
+    d = g.get_repo(dest_repo)
+
+    # Create list of labels to add/delete
+    src_labels = list(s.get_labels())
+    src_label_names = set([j.name for j in src_labels])
+    dest_labels = list(d.get_labels())
```

Comment:
```suggestion
    dst_labels = list(d.get_labels())
```

I like that because of the symmetry in length with `src`.